scripts/velocity.py

```diff
@@ -0,0 +1,382 @@
+            'node_id': node_id,
+            'name': github.get_name_by_repo_id(node_id)
+        }
+        config.workspace[ws['workspace_name']]['repos'].append(repo)
+    config.save()
+    print(f"\nConfigured with workspace `{ws['workspace_name']}`, connected to "
+          f"{len(ws['repo_ids'])} repositories. Configuration saved to {config.path}.")
+
+
+def report(workspace: str, since: str, until: str):
+    zenhub = ZenHubAPI(config.auth['zenhub_key'])
+    github = GitHubAPI(config.auth['github_user'], config.auth['github_key'])
+    repos = [repo['name'] for repo in config.workspace[workspace]['repos']]
+    payload = copy_json(github.get_issues_for_repos(since, until, repos))
+    issues = []
+    prs = []
+    for query_id, results in payload['data'].items():
+        for node in results['nodes']:
+            if node['__typename'] == 'PullRequest':
+                prs.append(PullRequest.from_dict(node, zenhub))
+            elif node['__typename'] == 'Issue':
+                issues.append(Issue.from_dict(node, zenhub))
+    points = sum(i.estimate for i in (issues + prs) if isinstance(i.estimate, int))
+    print(f"Report for {workspace=}: {since} - {until}")
+    print(f"Story points in this period: {points}")
+    sum_time = collections.Counter()
+    for pr in prs:
+        print(f"PR #{pr.number}: {pr.title} had {pr.reviews} reviews and "
+              f"{pr.lines_changed} lines changed ({pr.line_to_review_ratio})")
+        sum_time.update(pr.reviewer_time)
+    print(sum_time)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--configure',
+                        default=False,
+                        required=False,
+                        action='store_true',
+                        help="Configure credentials and repositories.")
+    parser.add_argument('workspace',
+                        nargs='?',
+                        help="ZenHub workspace to generate a report for.")
+    parser.add_argument('--since', '-s',
+                        default=(datetime.now() - timedelta(days=7)).strftime('%Y-%m-%d'),
+                        required=False,
+                        action='store',
+                        help="Grab issues that were closed after this date. "
+                             "Defaults to one week ago. (e.g. '2020-06-21')")
+    parser.add_argument('--until', '-u',
+                        default=datetime.now().strftime('%Y-%m-%d'),
+                        required=False,
+                        action='store',
+                        help="Grab issues that were closed before this date. "
+                             "Defaults to today. (e.g. '2020-06-30')")
+    arguments = parser.parse_args()
+
+    if arguments.configure:
+        configure()
+        exit(0)
+    elif arguments.workspace and not os.path.exists(DEFAULT_CONFIG_PATH):
+        print(f"Missing credentials: do `{sys.argv[0]} --configure`")
+        exit(1)
```

Comment:
Add logging (see: https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#11logging) and use `log.error()` instead of `print()` and `exit()`